src/api/app/controllers/source_controller.rb

```diff
@@ -996,6 +996,9 @@ def package_command_release
       multibuild_container = params[:package].gsub(/^.*:/, '')
     end
 
+    # uniq timestring for all targets
+    time_now = Time.now.utc
```

Comment:
why are you assigning it here if you only use it inside release_target_name in one line? :confused:

Reply:
I think it's inside a loop iteration and because he wants to have all release targets with the same time, IMO is ok. Am I right @adrianschroeter ?